examples/TestAppUwp/MainPage.xaml.cs

```diff
@@ -551,6 +555,10 @@ private async void OnLoaded(object sender, RoutedEventArgs e)
         {
             LogMessage("Initializing the WebRTC native plugin...");
 
+            // Cannot run in UI thread on UWP because this will initialize the global factory
+            // (first library call) which needs to be done on a background thread.
```

Comment:
nit: not for this review, but this looks like a very easy pitfall, it might be better to have an explicit initialization call instead.

Reply:
Yes, although explicit call is quite tricky in Unity